src/components/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,11 @@
+import './LandingPage.css';
+
+
+function LandingPage() {
+    return (
+        <section className="landingWrapper">
+            <div>Capstone DONE :: Mic Drop</div>
```

Comment:
Done!

Reply:
> We have a dashboard! I'll add a couple of tickets with specific next steps (functionality for various buttons, and small spelling error - thing `vender` should be `vendor`

I think both of these spellings are technically correct. We should definitely make sure to establish which one we want to use and make sure it's a norm across the board. Good call, Hanna!